projects/pytorch-py/Dockerfile

```diff
@@ -0,0 +1,56 @@
+# Copyright 2023 ISP RAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM sydr/ubuntu20.04-sydr-fuzz
+
+# Install build dependencies.
+RUN pip3 install astunparse numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions
+
+# Clone target from GitHub.
+RUN git clone https://github.com/pytorch/pytorch/ /pytorch
+RUN cd /pytorch && \
+    git checkout 22e2fd554cf370765d4c44fe2b99c8bb6e42b0bb && \
```

Comment:
Select PyTorch version to be compatible with vision and text

Reply:
torch text still fails, so returned `|| true` back